### ISSUE TITLE
orderBy is not working when we have 2 dots (.) in the field

### ISSUE THREAD

**Issue by MitulGedeeya** [Opteo/google-ads-api#300]
For Example: 

If I want to orderBy `ad_group_criterion.keyword.match_type` then it won't work.

I checked code of library and i found that following function has an issue

File: query.js

```
function completeOrderly(orderly, entity) {
    if (!orderly.length) {
        throw new Error(exports.QueryError.INVALID_ORDERLY);
    }
    else if (new RegExp(/^.[^.]+\..[^.]+$/g).test(orderly)) {
        // text before and after a full stop (e.g. campaign.resource_name)
        return orderly;
    }
    else if (new RegExp(/^.[^.]+$/g).test(orderly)) {
        // text without a full stop (e.g. resource_name)
        return entity + "." + orderly;
    }
    else {
        throw new Error(exports.QueryError.INVALID_ORDERLY);
    }
}
```

**Comment by MitulGedeeya** [comment]
@kritzware  @WillCooter we are on critical situation as our whole system has been broken. So, it's workaround I fixed in my local as follow :

`function completeOrderly(orderly, entity) {
    if (!orderly.length) {
        throw new Error(exports.QueryError.INVALID_ORDERLY);
    }
    else if (new RegExp(/^.[^.]+\.([^.]+\.)?.[^.]+$/g).test(orderly)) {
        // text before and after a full stop (e.g. campaign.resource_name)
        return orderly;
    }
    else if (new RegExp(/^.[^.]+$/g).test(orderly)) {
        // text without a full stop (e.g. resource_name)
        return entity + "." + orderly;
    }
    else {
        throw new Error(exports.QueryError.INVALID_ORDERLY);
    }
}`

**Comment by WillCooter** [comment]
@MitulGedeeya Well spotted, thanks for taking the time to make a new regex! I made one small adjustment to include any number of layers, for resource names such as `ad_group_ad.ad.gmail_ad.teaser.logo_image`. The final regex is `/^[^.\s]+(\.[^.\s]+)+$/g` and will be included in the next release (hopefully this week).

**Comment by WillCooter** [comment]
@MitulGedeeya  Thanks for your patience. This has been fixed in the latest version of the library (v8.0.0).
